.github/workflows/check_env.yaml

```
@@ -7,9 +7,9 @@ on:
 # Global environment variables that might need adaption in the future
 env:
   BACKEND: pytorch
-  ENV_NAME: CompSim-PINN
+  ENV_NAME: CompSim_PINNs
```

of course if we change the name of the environment, then we also have to adapt it here:
```suggestion
  ENV_NAME: compsim_pinns
```